src/common/components/Alert/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ReactNode } from 'react'
```

Comment:
@muhadovic change the file name to just Like "Alert.tsx"

Reply:
@dbizhev I follow the title of the design for filename. It it is wrong, please suggest the filename